.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,14 @@
+name: Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: swift build
+    - name: Test
+      run: swift test
```

Comment:
There is no SPM for Tempura for the reasons highlighted in #74, so this should change a little bit to do:
- `tuist generate`
- `pod install`
- `xcodebuild build-for-testing`
- `xcodebuild test-without-building`